corehq/form_processor/document_stores.py

```diff
@@ -21,14 +22,17 @@ def __init__(self, domain, xmlns=None):
 
     def get_document(self, doc_id):
         try:
-            return self.form_accessors.get_form(doc_id).to_json(include_attachments=True)
+            form = self.form_accessors.get_form(doc_id)
+            if isinstance(form, XFormInstanceSQL):
+                return form.to_json(include_attachments=True)
+            else:
+                return form.to_json()
```

Comment:
This is a little nasty. pulled it from https://github.com/dimagi/commcare-hq/pull/18456/files#diff-551bb43505398f33e4901de767d67560R56

@esoergel let me know if you came up with a utility to do it or decide on a better way

Reply:
Nope, just left it like that.  I did have two thoughts that I'd be in favor of, though:

We currently insert a field called `backend_id` (either `'couch'` or `'sql'`) before sending to to elasticsearch.  This could be added as a class attribute to `XFormInstance` and `XFormInstanceSQL`.

More specific to this issue, we could also override `to_json` on `XFormInstance` (right now I believe it's just the default `Document` implementation) and have it accept and ignore `include_attachments`.  It's kinda smelly to have a parameter that doesn't do anything, but I think it's less smelly than presenting a semi-opaque interface with params that only work sometimes (We similarly add and  ignore the `domain` parameter to make `CommCareUser` act like `WebUser`).